administrator/components/com_media/package.json

```diff
@@ -10,7 +10,9 @@
     "build-js": "browserify ./resources/scripts/mediamanager.js > ../../../media/com_media/js/mediamanager.js"
   },
   "dependencies": {
+    "amdefine": "^1.0.1",
     "babel-polyfill": "^6.23.0",
+    "has-flag": "^2.0.0",
```

Comment:
i could not do npm run build without those, i told you before about this issue and also @dgt41